lcls-twincat-common-components/Library/Library.plcproj

```diff
@@ -17,7 +17,7 @@
     <CombineIds>false</CombineIds>
     <Title>lcls-twincat-common-components</Title>
     <Released>false</Released>
-    <ProjectVersion>0.0.0</ProjectVersion>
+    <ProjectVersion>3.8.0</ProjectVersion>
```

Comment:
It's strange, but our release procedure currently expects these to be 0.0.0 at all non-tags and then changes them to specific numbers at tag time. So they need to go back to 0.0.0.
```suggestion
    <ProjectVersion>0.0.0</ProjectVersion>
```